src/store.c

```diff
@@ -185,12 +220,12 @@ struct rill_store *rill_store_open(const char *file)
     }
 
     if (!is_supported_version(store->head->version)) {
-        rill_fail("invalid version '%du' for '%s'", store->head->version, file);
+        rill_fail("invalid version '%zu' for '%s'", (uint64_t) store->head->version, file);
```

Comment:
I would not cast the value here. I would instead use the correct storage identifier in the fail statement. So I believe the original error was that an extra u appeared for no reason. I believe the correct storage identifier here is just `%u` although if you want to extremely pedantic you could use one of the fancy macros but that's a bit overkill.

Reply:
Weird, I don't remember why I did this. 'u' works fine without weirdness. Thanks.